External/FEXCore/Source/Interface/Core/OpcodeDispatcher.h

```diff
@@ -1040,19 +1060,105 @@ friend class FEXCore::IR::PassManager;
   [[nodiscard]] uint32_t GetDstBitSize(X86Tables::DecodedOp Op) const;
   [[nodiscard]] uint32_t GetSrcBitSize(X86Tables::DecodedOp Op) const;
 
+  static inline constexpr unsigned IndexNZCV(unsigned BitOffset) {
+    switch (BitOffset) {
+      case FEXCore::X86State::RFLAG_OF_LOC: return 28;
+      case FEXCore::X86State::RFLAG_CF_LOC: return 29;
+      case FEXCore::X86State::RFLAG_ZF_LOC: return 30;
+      case FEXCore::X86State::RFLAG_SF_LOC: return 31;
+      default: assert(0);
```

Comment:
This should be `FEX_UNREACHABLE` instead of assert, to remove the compiler warning.